plugins/fileservices/fileservices.cpp

```diff
@@ -3140,3 +3140,56 @@ FILESERVICES_API void FILESERVICES_CALL fsClearExpireDays(ICodeContext * ctx, co
     else
         throw makeStringExceptionV(0, "ClearExpireDays: Could not find logical file %s", lfn.str());
 }
+
+bool getDefaultValue(const char * processName, const char * propertyName, StringBuffer & strDefaultValue)
+{
+    // Returns with true if a default value retrieved and false if not.
+
+    #ifdef _CONTAINERIZED
```

Comment:
style: we don't normally indent #if's/#endif's

Reply:
Changed